assets/src/components/ladderPage.tsx

```diff
@@ -64,8 +76,24 @@ const LadderTab = ({
       onClick={() => selectTab()}
     >
       <div className="m-ladder-page__tab-contents">
-        <div className="m-ladder-page__tab-title">{title}</div>
-        {tab.isCurrentTab ? saveIcon("m-ladder-page__tab-save-icon") : null}
+        <div
+          className={
+            "m-ladder-page__tab-title" +
+            (tab.saveChangesToTabUuid ? " m-ladder-page__tab-title-edited" : "")
```

Comment:
```suggestion
            (tab.saveChangesToTabUuid ? " m-ladder-page__tab-title--edited" : "")
```

**nitpick:** should this be a double-dash before the modifier? (I'm rusty on my BEM and it never stopped being at least a little confusing for me, so I might well be wrong. )

Reply:
Good question, I was just poking through the BEM documentation and I believe you're right. I'll admit I haven't been the most consistent about making sure we're doing BEM naming by the book, so this gives me another bit of motivation to go through and clean up our class naming conventions at some point.